nixos/modules/security/ipa.nix

```
@@ -226,58 +228,61 @@ in
             4. Restart sssd systemd service: sudo systemctl restart sssd
 
         EOF
+        # let service fail, to raise awaitness
```

Is this what you meant?

```suggestion
        # let service fail, to raise awareness
```